test/maps/all_tribes.wmf/scripting/init.lua

```diff
@@ -164,7 +174,7 @@ end
 function place_player_ship(playernumber)
    local player = wl.Game().players[playernumber]
    local starting_field = wl.Game().map.player_slots[playernumber].starting_field
```

Comment:
You are right, I forgot to change this line.

I is in a commit of a coming pull request. :see_no_evil: